pb/cmd_pb/cmd.proto

```diff
@@ -97,7 +97,8 @@ message Router {
   bytes fingerprint = 3;
   uint32 cost = 4;
   bool noTraversal = 5;
-  map<string, TagValue> tags = 6;
```

Comment:
Re-ordering fields breaks backwards compat, no?

Reply:
Correct. However, not going to worry about it until 1.0 release, as there are no current install using raft.